frontend/src/api/game.ts

```
@@ -97,11 +111,64 @@ export const getUserItems = async (email: string): Promise<GameItemsResponse> =>
         const response = await api.post(`/items/use/${itemIdx}`, null, {
-            params: { email },
+            headers: { email },
+        });
+        return response.data;
+    } catch (error) {
+        throw error;
+    }
+};
+
+// 아이템 장착
+export const equipItem = async (email: string, itemIdx: number, slot: number): Promise<UserItemDTO> => {
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)